chapter10/mssw/exercise2.pl

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Data::Dumper;
+use Benchmark qw( :all );
+
+sub human_friendly_sort {
+    my @sorted = sort( { -s $a <=> -s $b } glob('*') );
+}
+
+sub cyborg_friendly_sort {
+    my @sorted =
+      map( $_->[0],
+        sort( { $a->[1] <=> $b->[1] } map( [ $_, -s $_ ], glob('*') ) ) );
+}
```

Comment:
i was just trying to understand what was being done when!
i started by dumping glob, then dumping the mapped glob, then the sorted mapped glob, then ended up with the mapped sorted mapped glob. So i added brackets as i was building it up.